common/web/gesture-recognizer/src/inputEventEngine.ts

```diff
@@ -18,20 +24,77 @@ namespace com.keyman.osk {
     abstract registerEventHandlers();
     abstract unregisterEventHandlers();
 
-    onInputStart(coord: InputEventCoordinate) {
-      this.emit(InputEventEngine.INPUT_UPDATE_EVENT_NAME, TrackedInputState.START, coord);
+    /**
+     * @param identifier The identifier number corresponding to the input sequence.
+     */
+    hasActiveSequence(identifier: number) {
+      return this._activeSequenceWrappers.findIndex((seq) => seq.item.identifier == identifier) != -1;
+    }
+
+    private getSequenceWrapperWithId(identifier: number) {
+      return this._activeSequenceWrappers.find((seq) => seq.item.identifier == identifier);
+    }
+
+    public cleanupSequenceWithId(identifier: number) {
```

Comment:
What is the purpose of this function -- from the name, it sounds like it looks for a sequence with the given id `identifier` and "cleans it up" ... whatever that means? 😉 

Could we rename it to something more like `removeMatchingSequence`?

Reply:
Note line 17 - this class tracks all sequences that are expecting updates.  This method exists to say "We're no longer expecting updates here - stop all tracking.  And cancel all event handlers for updates on the sequence, too."  (The latter part is for proactive memory-leak prevention.)

So, maybe 'finalize', or 'stopTracking'?